sdk/python/tests/compiler/testdata/uri_artifacts.py

```diff
@@ -54,7 +54,7 @@ def get_code(self, ):
 read_from_gcs = components.load_component_from_text("""
 name: Read from GCS
 inputs:
-- {name: input_gcs_path, type: String, description: 'GCS file path'}
+- {name: input_gcs_path, type: Artifact, description: 'GCS file path'}
```

Comment:
I agree that the Artifact type is better fitting here.
Let's change the input name and description accordingly. Otherwise the input name does not seem to correspond to the type.
```
- {name: artifact, type: Artifact, description: 'Some artifact'}
```
Then we change the confusing `{inputUri: input_gcs_path}` (which reads as "The URI of the input_gcs_path contents" = "The URI of a path string" which is pretty weird) to `{inputUri: artifact}` (which reads as "The URI of the input artifact").